apiClient.go

```diff
@@ -15,12 +16,13 @@ import (
 	"github.com/sethgrid/pester"
 )
 
-type ApiClient interface {
-	GetStatus(ctx context.Context, repoSource, repoOwner, repoName string) (status string, err error)
+type SnykApiClient interface {
+	OrganizationProjects(ctx context.Context, orgID string) (Projects, error)
+	ProjectVulnerabilities(ctx context.Context, orgID, projectID string, filters *projectFilters) (Vulnerabilities, error)
 }
 
-// NewApiClient returns a new ApiClient
-func NewApiClient(apiToken string) ApiClient {
+//NewApiClient returns a new SnykApiClient
+func NewApiClient(apiToken string) SnykApiClient {
```

Comment:
Best to rename the func to `NewSnykAPIClient` and `SnykApiClient` to `SnykAPIClient` (golang expects acronyms like API to be written in uppercase).